### PR TITLE
Fix dateutil import

### PR DESCRIPTION
**Pull request by mkurek**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -31,28 +31,30 @@
...
-        'djangorestframework==2.4.3',
         'django-rq==0.4.5',
-        'MySQL-python==1.2.3',
-        'rq>=0.3.7',
-        'dj.choices==0.9.2',
-        'SQLAlchemy==0.7.8',
+        'django-simple-history==1.6.3',
+        'django==1.4.22',
+        'djangorestframework==2.4.3',
+        'djangorestframework==2.4.3',
```
Comment (szok) [inline setup.py]: duplicate :)

File: src/ralph_scrooge/settings/prod.py [hunk 2]
```diff
@@ -1,5 +1,12 @@
+import os
+
 from ralph_scrooge.settings.base import *  # noqa
 
+MEDIA_ROOT = os.path.expanduser('~/.scrooge/shared/uploads')
+STATIC_ROOT = os.path.expanduser('~/.scrooge/shared/static')
+
+INSTALLED_APPS += ('gunicorn',)
```
Comment (xor-xor) [inline src/ralph_scrooge/settings/prod.py]: Flake has some problem with this (see error from Travis).

Comment (mkurek) [reply src/ralph_scrooge/settings/prod.py]: fixed

File: setup.py (hunk 1 above)
Comment (mkurek) [reply setup.py]: done

**Review by szok — APPROVED** [review]

**Comment by xor-xor** [comment]
:ok:

**Pull request merged**